packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -216,6 +217,7 @@ export class CorrelatedSceneGraph {
   private[$gltf]: GLTF;
   private[$gltfElementMap]: GLTFElementToThreeObjectMap;
   private[$threeObjectMap]: ThreeObjectToGLTFElementHandleMap;
+  private[$gltfVariantMap]: GLTFElementToThreeObjectMap|null = null;
```

Comment:
Created a variant map to hold glTF elements belonging to a variant definition. This prevents them from being all being correlated to the default material variant.